tee-worker/litentry/core/data-providers/src/achainable.rs

```diff
@@ -96,8 +97,13 @@ impl AchainablePost for AchainableClient {
 		let response = self
 			.client
 			.post_capture::<SystemLabelReqPath, ReqBody, serde_json::Value>(params, body);
-		debug!("ReqBody response: {:?}", response);
-		response.map_err(|e| Error::AchainableError(format!("Achainable response error: {}", e)))
+		match response {
+			Ok(r) => {
+				debug!("ReqBody response: {}", r);
```

Comment:
do we want to keep this log? seems noisy to me, but ok if we find it valuable

Reply:
Yes only in dev - never in prod